memgpt/local_llm/llm_chat_completion_wrappers/airoboros.py

```diff
@@ -0,0 +1,146 @@
+import json
+
+from .wrapper_base import LLMChatCompletionWrapper
+
+
+class Airoboros21Wrapper(LLMChatCompletionWrapper):

```

Comment:
Should probably make it clear that this wrapper creates a prompt that's a "no inner thoughts" bot. Can revise the prompt later to make a inner thoughts + function call bot too (need to experiment).